app/controllers/credentials/credentials-form.js

```diff
@@ -32,6 +32,7 @@ class CredentialsForm {
           errors[field.id] = validator.message
           return true
         }
+        return false
```

Comment:
I'd strongly recommend sanity-checking this form and its test coverage, just to make sure there's no unexpected behaviour with this change. It might look innocuous (and may well be 100% fine!) but we've been bitten before in these scenarios.

Reply:
I've added a more comprehensive test, I'm 99% sure this should be fine because the callback provided to `Array.prototype.some` should return a boolean value, and returning nothing would be the same was returning `undefined`, which is falsy, so returning `false` instead shouldn't affect the functionality

Reply:
Fab, thank you 👍 

I will leave it to other folks to review the rest of the PR 😸